src/main/java/io/github/thebusybiscuit/mobcapturer/listeners/PelletListener.java

```diff
@@ -67,15 +68,24 @@ && canCapture(player, entity)
      */
     @ParametersAreNonnullByDefault
     protected boolean canCapture(Player p, LivingEntity entity) {
-        if (Slimefun.getProtectionManager().hasPermission(p, entity.getLocation(), Interaction.ATTACK_ENTITY)) {
-            if (MobCapturer.getRegistry().getConfig().getBoolean("options.capture-named-mobs")) {
-                return true;
-            } else {
-                return entity.getCustomName() == null;
-            }
-        } else {
+
```

Comment:
remove this empty line

Reply:
There is no empty line anymore, I have reworked the method to use return statements instead of else

Reply:
I still see an empty line here

Reply:
I removed some line, if this is still not the correct line then please remove it yourself